.gitignore

```diff
@@ -1 +1,2 @@
 .creds
+.DS_Store
```

Comment:
Ignoring files that are not a part or dependency of the repo should be done in a global ignore instead:

https://github.com/alphagov/govuk-developer-docs/commit/918783c8655ae067f9f3751853a23d97d70fec98